<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 202, 18);</string>
    </property>
    <property name="title">
     <string>To Do List</string>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Deadline:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add1">
     <property name="geometry">
      <rect>
       <x>554</x>
       <y>150</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Task</string>
     </property>
    </widget>
    <widget class="QPushButton" name="done1">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>150</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Deadline:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Task</string>
     </property>
    </widget>
    <widget class="QPushButton" name="done2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Deadline:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>390</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Task</string>
     </property>
    </widget>
    <widget class="QPushButton" name="done3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>390</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Deadline:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add4">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>510</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Task</string>
     </property>
    </widget>
    <widget class="QPushButton" name="done4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>510</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>560</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Project by Rohan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 26, 255);
font: 75 24pt &quot;Times New Roman&quot;;
color: red;</string>
     </property>
     <property name="text">
      <string>TO DO LIST</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="task1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>721</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="task2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>721</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="task3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>721</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="task4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>721</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="deadline1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="deadline2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="deadline3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>380</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="deadline4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <zorder>task4</zorder>
    <zorder>task3</zorder>
    <zorder>task2</zorder>
    <zorder>task1</zorder>
    <zorder>label_1</zorder>
    <zorder>add1</zorder>
    <zorder>done1</zorder>
    <zorder>label_2</zorder>
    <zorder>add2</zorder>
    <zorder>done2</zorder>
    <zorder>label_3</zorder>
    <zorder>add3</zorder>
    <zorder>done3</zorder>
    <zorder>label_4</zorder>
    <zorder>add4</zorder>
    <zorder>done4</zorder>
    <zorder>label</zorder>
    <zorder>label_5</zorder>
    <zorder>deadline1</zorder>
    <zorder>deadline2</zorder>
    <zorder>deadline3</zorder>
    <zorder>deadline4</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
